src/account_management.rs

```
@@ -0,0 +1,267 @@
+    format!("{ACCOUNTS_DIR}/{account_name}/store.db")
+}
+
+pub fn ensure_accounts_dir() -> Result<()> {
+    if !Path::new(ACCOUNTS_DIR).exists() {
+        fs::create_dir_all(ACCOUNTS_DIR)?;
+    }
+    Ok(())
+}
+
+pub fn list_accounts() -> Result<Vec<String>> {
+    ensure_accounts_dir()?;
+    let mut accounts = Vec::new();
+
+    if let Ok(entries) = fs::read_dir(ACCOUNTS_DIR) {
+        for entry in entries {
+            if let Ok(entry) = entry
+                && entry.path().is_dir()
+                && let Some(name) = entry.file_name().to_str()
+            {
+                let store_path = get_account_store_path(name);
+                if Path::new(&store_path).exists() {
+                    accounts.push(name.to_string());
+                }
+            }
+        }
+    }
+
+    Ok(accounts)
+}
+
+/// Creates new manager for specific account
+pub async fn create_registered_manager_for_account(
+    account_name: &str,
+) -> Result<Manager<SqliteStore, Registered>> {
+    let store_path = get_account_store_path(account_name);
+    let store = open_store(&store_path).await?;
+
+    match Manager::load_registered(store).await {
+        Ok(manager) => Ok(manager),
+        Err(err) => Err(Error::new(err)),
+    }
+}
+
+pub async fn cleanup_invalid_accounts() -> Result<Vec<String>> {
+    let mut invalid_accounts = Vec::new();
+    let accounts = list_accounts()?;
+
+    for account_name in accounts {
+        let store_path = get_account_store_path(&account_name);
+        match open_store(&store_path).await {
+            Ok(store) => match Manager::load_registered(store).await {
+                Ok(_) => {}
+                Err(_) => {
+                    invalid_accounts.push(account_name.clone());
+                    let account_dir = format!("{ACCOUNTS_DIR}/{account_name}");
+                    if Path::new(&account_dir).exists() {
+                        let _ = std::fs::remove_dir_all(&account_dir);
```

Please, don't ignore errors explicitly. At least log them.